src/components/Sections/SectionList.js

```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from 'react';
```

Comment:
isn't this a table with two columns?

Reply:
Yes, implemented the same but used as a list.